siatest/consensus/consensus_test.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/node"
+	"github.com/NebulousLabs/Sia/siatest"
+)
+
+// TestApiHeight checks if the consensus api endpoint works
+func TestApiHeight(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	testdir, err := siatest.TestDir(siatest.SiaTestingDir, t.Name())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Create a new server
+	c, s, err := siatest.NewClientServerPair(node.AllModules(testdir))
```

Comment:
Instead of returning a `c` and an `s` separately, it should return a `TestNode` object which has a `c` and an `s` as public fields.

Reply:
oh hmm, or maybe they should just be embedded fields, that might be even easier.